### PR TITLE
Updating Dice rolls info

### PR DESCRIPTION
**Pull request by Masterdiekillsu**
Lines 34,35,36, and 37 have been edited to reflect the new changes to combat and the game.

### CODE REVIEW

**Review by rosebrallier — APPROVED** [review]
Thanks Nate!
Non-blocking tone note: I think the tone would be more effective if we deleted the "Now" at the top of the paragraph.

**Review by bleehu — APPROVED** [review]
Shipit!

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -31,9 +31,11 @@ get from the dice. This represents how good or bad your particular character is
 at doing whatever it is you are attempting to do. What to add when is discussed
 in detail below. 
 
-	For now, know that Compound X uses two ten-sided dice (usually called D10s) to
-represent a d100, a percent. One of the D10s represents the number in the 10's place.
-the other represents the number in the 1's place. Most dice sets come with two
+	Now, there will be two kinds of rolls your character will be making. 
+A skill roll, and a combat roll. Skill rolls use two ten-sided die, to represent
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt:35]: two ten-sided ^dice^

File: 01_01_Basic_Rules.txt [hunk 2]
```diff
@@ -31,9 +31,11 @@ get from the dice. This represents how good or bad your particular character is
 at doing whatever it is you are attempting to do. What to add when is discussed
 in detail below. 
 
-	For now, know that Compound X uses two ten-sided dice (usually called D10s) to
-represent a d100, a percent. One of the D10s represents the number in the 10's place.
-the other represents the number in the 1's place. Most dice sets come with two
+	Now, there will be two kinds of rolls your character will be making. 
+A skill roll, and a combat roll. Skill rolls use two ten-sided die, to represent 
+a d100, which in turn represents a percentage. One die is designated as the tens 
+place, and the other die as the ones place. However, in combat, you will use one d10,
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt:37]: May be worth it to specify that the tens and ones place dice are chosen before the roll.

File: 01_01_Basic_Rules.txt [hunk 3]
```diff
@@ -31,9 +31,11 @@ get from the dice. This represents how good or bad your particular character is
...
 in detail below. 
 
-	For now, know that Compound X uses two ten-sided dice (usually called D10s) to
-represent a d100, a percent. One of the D10s represents the number in the 10's place.
-the other represents the number in the 1's place. Most dice sets come with two
+	Now, there will be two kinds of rolls your character will be making. 
+A skill roll, and a combat roll. Skill rolls use two ten-sided die, to represent 
+a d100, which in turn represents a percentage. One die is designated as the tens 
+place, and the other die as the ones place. However, in combat, you will use one d10, 
+which is reflected in a single number system.  Most dice sets come with two
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt:38]: Digit, not number.

**Review by Turtlelord26 — APPROVED** [review]
Nonblocking grammar notes inline.

**Pull request merged**